### ISSUE TITLE
failed to install Module::Build::Tiny

### ISSUE THREAD

**Issue by skaji** [057a3dd61f99517a3afea0051a49cb27994f94d/rslinux#1]
My environment
```
❯ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04 LTS"

❯ perl -v
This is perl 5, version 26, subversion 2 (v5.26.2) built for x86_64-linux

❯ perl -MApp::rs\ 99999
App::rs version 99999 required--this is only version v2.1.1.
```

Then `rs install Module::Build::Tiny` failed.
The whole output is https://gist.github.com/skaji/91c59ee1f1fb0580b6e81e0326bb98ea

**Comment by skaji** [comment]
Ah, rs assumes I set PERL5LIB=/home/skaji/CPAN/lib/perl5.

**Comment by 057a3dd61f99517a3afea0051a49cb27994f94d** [comment]
@skaji Thanks for the report.
Actually I intend to set the environment for the user and provide a hint if it's not set already, but I used PERLLIB instead of PERL5LIB, the priority of which is lower and got overridden.
Now fixed in 626c2d86b0a1d9a9306d11dbd9fcaf07a4fd7003 where PERL5LIB is used, I tried to install Module::Build::Tiny and succeeded.
